addons/website/views/website_templates.xml

```diff
@@ -1406,7 +1406,7 @@
 <template id="template_footer_descriptive" inherit_id="website.layout" name="Descriptive" active="False">
     <xpath expr="//div[@id='footer']" position="replace">
         <div id="footer" class="oe_structure oe_structure_solo" t-ignore="true" t-if="not no_footer">
-            <section class="s_title pt48 pb24">
+            <section class="s_title pt48 pb24" vcss="001">
```

Comment:
Shouldn't this be a stable fix?

Reply:
I opened a new PR here https://github.com/odoo/odoo/pull/75594, 
this PR's test will fail until it is forwarded to master